### ISSUE TITLE
The add-on won't work?

### ISSUE THREAD

**Issue by BlobPls** [3DMish/Blender-Add-ons-3DM-Snow#4]
I installed the add-on but when I check prefrences and install add-ons from file I don't see it in prefrences nor do I see anything

**Comment by tibfox** [comment]
Yeah me too.. I don't know how to activate it after installing without any error :/ @3DMish could/would you help us please ? :)

**Comment by tibfox** [comment]
okay I got it 👯‍♂ 
extract the archive and copy the "3dm_snow_for_2_80.py" file into yourblenderpath\scripts\addons
after that, start blender, go in preferences and activate it :)
